internal/colorize/wrap.go

```diff
@@ -100,6 +115,19 @@ func inRange(pos int, ranges [][]int) bool {
 	return false
 }
 
+// colorTag returns the currently active color tag (if any) at the given position.
+func colorTag(pos int, ranges [][]int, names [][]string) string {
```

Comment:
This assumes there's only ever one color tag, but we could have: `[BOLD][RED]`, for example.